app/src/main/java/org/mozilla/focus/browser/cursor/CursorKeyDispatcher.kt

```diff
@@ -30,7 +30,7 @@ class CursorKeyDispatcher(
         val remoteKey = RemoteKey.fromKeyEvent(event)
         if (remoteKey == RemoteKey.CENTER ||
                 event.keyCode == KeyEvent.KEYCODE_ENTER) { // For keyboard and emulator use.
-            dispatchTouchEventOnCurrentPosition(event.action)
+            dispatchTouchEventOnCurrentPosition(event)
```

Comment:
Similarly, so I don't have to "review" these changes twice, it'd be ideal if they were in the first commit.